app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -31,6 +32,9 @@ function InPersonPrepareStep({ toPreviousStep }) {
       forceRedirect(previousStepURL);
     } else if (skipDocAuthFromHowToVerify && howToVerifyURL) {
       forceRedirect(howToVerifyURL);
+    } else if (skipDocAuthFromSocureHybrid) {
+      // directly from Socure Hybrid page
+      forceRedirect('https://google.com');
```

Comment:
left over form testing? 😅

Reply:
Heh... the `Back` link and the browser back button are not the same. Updated the feature specs to actually test the back link and did some back flips and twists to actually make them work in both hybrid and standard flows. Fixed in 50537f6bbb